src/shapes/Group.spec.ts

```diff
@@ -270,14 +342,14 @@ describe('Group', () => {
       });
       expect(group.top).toBe(20);
       expect(group.left).toBe(40);
-      // TO DO BUG: this should be 30
+      // TODO BUG: this should be 30
```

Comment:
renamed this from `TO DO` to `TODO` so it is properly highlighted in editors and recognized as a proper todo comment